### DISCUSSION TITLE
What's the difference between variable_ttf and ttf?

### DISCUSSION THREAD

**Discussion by just-a-hriday** [tonsky/FiraCode#1275] [Q&A] [answered]
I was going to install the font, but I noticed that there's a variable_ttf folder too, which has only one font inside it. What's the difference between that and the regular ttf folder?

**Comment by tonsky** [comment, marked as answer]
Variable is a dynamic font file that can extrapolate to any weight between Light and Bold. They don’t work all that well on Desktop (support is limited), but on the Web you can already use them. Will save you a few kilobytes as well

**Reply by wopian** [reply]
To expand on this, https://v-fonts.com/ and https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Fonts/Variable_Fonts_Guide give a great (interactive) overview of the capabilities of the Variable Font specification.

It can save a lot of kilobytes, as instead of loading in Thin, Regular, Medium, Bold (that say, weigh 20kb each for 80kb total) you load the single variable font (that would maybe be under 25kb)
